### DISCUSSION TITLE
troubleshooting tools

### DISCUSSION THREAD

**Discussion by l3iggs** [ChristopherKlay/StadiaEnhanced#203] [Ideas & Requests]
Sometimes it's a bit difficult to help troubleshoot issues with a person's setup/network/stream. I might ask them to send me a screenshot of the stream monitor sidebar and sometimes there's not enough info there to pinpoint the problem. Would be cool if there was a "troubleshooting report" button a person could press that generates a bunch of copy/pasteable text with whatever useful troubleshooting info you can gather in one place. Ideas are:
- browser name/version
- extension version
- os info
- GPU/decoder info
- hostname of Stadia stream server (allows for traceroute later)
- detected screen res
- some historical latency info
- everything in the stream info sidebar
...
There's a lot more that could go into such a report!

**Comment by ChristopherKlay** [comment]
I likely won't add this as a UI element (sadly, the Stadia menu is fairly crowded already), but adding it to the `debugEnhanced()` console command should be possible, so a simple copy/paste commmand gives you a logged table with most if not all of that data, including the current stream data if a game is running.

I'll add it to the to-do list, likely fairly high in the list (:
